modules/data.atmosphere/R/GEFS_helper_functions.R

```diff
@@ -222,23 +193,13 @@ process_gridded_noaa_download <- function(lat_list,
     dlwrfsfc <- array(NA, dim = c(site_length, length(hours_char)))
     dswrfsfc <- array(NA, dim = c(site_length, length(hours_char)))
     
-    if(ens_index == 1){
-      base_filename2 <- paste0("gec00",".t",cycle,"z.pgrb2a.0p50.f")
-    }else{
-      if(ens_index-1 < 10){
-        ens_name <- paste0("0",ens_index-1)
-      }else{
-        ens_name <- as.character(ens_index-1)
-      }
-      base_filename2 <- paste0("gep",ens_name,".t",cycle,"z.pgrb2a.0p50.f")
-    }
+    member_type <- if (ens_index == 1) { "gec" } else { "gep" } # "_c_ontrol", "_p_erturbed"
+    ens_idxname <- stringr::str_pad(ens_index - 1, width = 2, pad = "0")
+    base_filename2 <- paste0(member_type,ens_idxname,".t",cycle,"z.pgrb2a.0p50.f")
     
     lats <- round(lat_list/.5)*.5
     lons <- round(lon_list/.5)*.5
```

Comment:
Not for this PR, but: GEFS is now also available on a 0.25 degree grid. Would it be of interest to add support for that?